docs/conf.py

```diff
@@ -177,3 +188,49 @@
 
 # If false, no module index is generated.
 #latex_use_modindex = True
+
+# -- Options for manual page output ---------------------------------------
+
+# One entry per manual page. List of tuples
+# (source start file, name, description, authors, manual section).
+man_pages = [
+    ('index', 'z3cform', 'z3c.form Documentation',
```

Comment:
Here and below, there are a couple of `z3c.form` mentions that should be `Products.GenericSetup`.

Reply:
Sorry for copy-pasting without reading it through again.